### ISSUE TITLE
Doesn't compile with OCaml 4.06?

### ISSUE THREAD

**Issue by bcpierce00** [backtracking/bibtex2html#17]
Looks like maybe the makefile needs -unsafe-string now?

```
Warning 3: deprecated: String.create
# Use Bytes.create instead.
# File "biboutput.ml", line 170, characters 27-44:
# Warning 3: deprecated: String.unsafe_set
# File "biboutput.ml", line 173, characters 6-23:
# Warning 3: deprecated: String.unsafe_set
# File "biboutput.ml", line 175, characters 4-6:
# Error: This expression has type bytes but an expression was expected of type
#          string
# make: *** [biboutput.cmx] Error 2
```

**Comment by backtracking** [comment]
This should be fixed with commit 8f25afb95 (using Buffer instead of String).

**Comment by bcpierce00** [comment]
Thanks!
